galasa-ui/src/contexts/DateTimeFormatContext.tsx

```diff
@@ -78,32 +101,48 @@ export function DateTimeFormatProvider({ children }: { children: React.ReactNode
       }
 
       const { dateTimeFormatType, locale, timeFormat } = preferences;
-        
-      const options: Intl.DateTimeFormatOptions = {
+      const resolvedTimeZone = getResolvedTimeZone();
+
+      // Define options for the main date/time part
+      const dateTimeOptions: Intl.DateTimeFormatOptions = {
         year: "numeric",
         month: "2-digit",
         day: "2-digit",
         hour: "2-digit",
         minute: "2-digit",
         second: "2-digit",
         hour12: timeFormat === '12-hour',
+        timeZone: resolvedTimeZone,
       };
 
-      if (dateTimeFormatType === 'browser') {
-        // Pass undefined to use the browser's default locale
-        formattedDate = new Intl.DateTimeFormat(undefined, options).format(date);
-      } else {
-        // Use the custom locale
-        formattedDate = new Intl.DateTimeFormat(locale, options).format(date);
-      }
+      // Define options specifically to extract the timezone name
+      const timeZoneNameOptions: Intl.DateTimeFormatOptions = {
+        timeZone: resolvedTimeZone,
+        timeZoneName: "short", // e.g., PST, EDT, EEST
```

Comment:
What does the "short" mean ? should it be "utc" or something ?

Wondering now if someone just wants to set their preference to UTC un-adulterated dates. Is that something which is possible ?